clock_exercise/AlarmClockListPage.qml

```diff
@@ -0,0 +1,93 @@
+import QtQuick 2.0
+
+Rectangle {
+    id: alarmClockList
+
+    signal back
+
+    color: "#151B2E"
+
+    BackButton {
+        x: 28
+        y: 20
+        onBack: {
+            clockManager.alarmItemModel.setAllNotSelected()
+            alarmClockList.back()
+        }
+    }
+
+    TitleOfThePage {
```

Comment:
Una piccola nota sul naming, anche questo non serve cambiarlo adesso, te lo scrivo giusto per parlarne: quando si nominano variabili, classi o componenti (nel caso del QML) non si usano mai né articoli né preposizioni.
Mi riferisco in particolare ai due `Of` e `The` presenti nel nome della componente.
In questo caso la scelta più giusta per il nome della componente sarebbe stato probabilmente `PageTitle`, che è molto più sintetico come nome ma del tutto equivalente da punto di vista del significato.